custom-vu/src/main/kotlin/jces1209/vu/page/DcIssuePage.kt

```diff
@@ -52,6 +54,69 @@ class DcIssuePage(
             ExpectedConditions
                 .invisibilityOfAllElements(descriptionForm)
         )
-        return this;
+        return this
+    }
+
+    override fun linkIssueButton(): DcIssuePage {
+        driver
+            .wait(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[aria-controls='opsbar-operations_more_drop']")))
+            .click()
+
+        driver
+            .wait(ExpectedConditions.visibilityOfElementLocated(By.id("link-issue")))
+            .click()
+
+        driver
+            .wait(ExpectedConditions.visibilityOfElementLocated(By.id(linkIssueSearchDialogId)))
+
+        return this
+    }
+
+    override fun linkIssueSearchAndChoose(issuePrefix: String): DcIssuePage {
+        driver
+            .wait(ExpectedConditions
+                .visibilityOfElementLocated(By.id("remote-jira-issue-search")))
+            .click()
+
+        val searchButton = driver
+            .wait(ExpectedConditions
+                .visibilityOfElementLocated(By.id("simple-search-panel-button")))
+
+        Actions(driver)
+            .sendKeys(issuePrefix)
```

Comment:
So we assume that there will be another issue in this project.
It will most likely be true, but let's explicitly state this assumpting as a code comment, I think the `AbstractIssuePage.linkIssueSearchAndChoose` javadoc is the proper place.

- [ ] make assumptions explicit